scripts/configure-prerelease.mjs

```
@@ -0,0 +1,238 @@
+async function main() {
+  const args = process.argv.slice(2)
+  const command = args[0] || 'test'
+
+  try {
+    switch (command) {
+      case 'test':
+        console.log('🚀 Testing Pre-release Configuration\n')
+
+        const configValid = validatePreReleaseConfig()
+        const preReleaseValid = testChangesetPrerelease()
+        testVersionCommands()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)